### PR TITLE
Privacy notice

### PR DESCRIPTION
**Pull request by patrickjfl**
# Context

[Trello ticket](https://trello.com/c/qzRGzf4n/452-content-privacy-notice)

# Changes in this PR

## Screenshots of UI changes

Screenshot 2024-01-11 at 15 02 48

### Before

Screenshot 2024-01-11 at 15 03 56

### After

Screenshot 2024-01-11 at 15 03 30

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

**Review by libuk — APPROVED** [review]
Looks good!

File: server/views/static/privacy-notice.njk [hunk 1]
```diff
@@ -0,0 +1,217 @@
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Privacy notice" %}
```
[the remark this answers is not in the document]
Comment (patrickjfl) [reply server/views/static/privacy-notice.njk:3]: ah yes, thanks Dan

File: server/routes/static.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export default function staticRoutes(controllers: Controllers, router: Router, s
   const { staticController } = controllers
 
   get(paths.static.maintenancePage.pattern, staticController.maintenancePage(), {})
+  get(paths.static.privacyNotice.pattern, staticController.privacyNoticePage(), {})
```
Comment (Harriethw) [inline server/routes/static.ts:16]: @tahb @edavey do you think we need an audit event for viewing this privacy page?  Or are the audit logs more for a trail to see what data a user has accessed?

**Review by Harriethw — APPROVED** [review]
Looks great! Just have that one question that Tom or Ed might know answer to.

File: server/routes/static.ts (hunk 2 above)
Comment (edavey) [reply server/routes/static.ts:16]: no I don't think it's appropriate to record viewing the privacy page in the HMPPS Audit service. 

That service existis to provide an audit log of who has attempted to view what, from a security, authorisation and need-to-view perspective. The Audit service is about understanding whether access to controlled data is being abused or violations to access control are being attempted.

**Pull request merged**